### PR TITLE
Env variables

### PR DESCRIPTION
**Pull request by SamLee514**
# Summary

Moving sentry url from strings.json to environment variable after I had been unknowingly spamming vaken status for the past month and a half.

Changes GCP bucket template string into one-liner as per README

Remove contributoors, replace with VandyHacks

# Review Focus 

Anything you want the reviewer to pay attention to? Feel free to delete this section.

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -21,18 +21,7 @@ GITHUB_CLIENT_ID=test
...
-  "private_key_id": "aprivatekey", \
-  "private_key": "anotherprivatekey", \
-  "client_email": "aserviceaccountemail@example.com", \
-  "client_id": "aclientid", \
-  "auth_uri": "https://accounts.google.com/o/oauth2/auth", \
-  "token_uri": "https://oauth2.googleapis.com/token", \
-  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs", \
-  "client_x509_cert_url": "acerturl" \
-}'
+GCP_STORAGE_SERVICE_ACCOUNT='{"type":"service_account","project_id":"foo","private_key_id":"aprivatekey","private_key": "anotherprivatekey","client_email": "aserviceaccountemail@example.com","client_id": "aclientid","auth_uri": "https://accounts.google.com/o/oauth2/auth","token_uri": "https://oauth2.googleapis.com/token","auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs","client_x509_cert_url": "acerturl"}'
```
Comment (bencooper222) [inline .env.template:24]: Probably better to break this up? Makes it easier to read?

**Review by bencooper222 — APPROVED** [review]

File: .env.template (hunk 1 above)
Comment (leonm1) [reply .env.template:24]: Perhaps leave a comment above it reminding whoever uses the template to remove the spaces?

File: src/client/index.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import * as Sentry from '@sentry/browser';
 import Vaken from './app';
 import STRINGS from './assets/strings.json';
 
-Sentry.init({ dsn: STRINGS.SENTRY_URL, tracesSampleRate: 1.0 });
+Sentry.init({ dsn: process.env.SENTRY_URL, tracesSampleRate: 1.0 });
```
Comment (leonm1) [inline src/client/index.tsx:9]: This won't actually work once it gets to production, as process.env doesn't exist on the client side. You can use the [webpack definePlugin](https://webpack.js.org/plugins/define-plugin/) instead to define SENTRY_URL as a build time constant, and process.env _will_ be defined for the webpack.config.js files.

So:
```
// webpack.something.js
new webpack.DefinePlugin({
  "process.env.SENTRY_URL": JSON.stringify(process.env.SENTRY_URL),
  // ALTERNATIVELY:
  SENTRY_URL: JSON.stringify(process.env.SENTRY_URL), // and access with just the global constant SENTRY_URL
});
```

**Review by leonm1 — CHANGES_REQUESTED** [review]

File: src/client/index.tsx (hunk 2 above)
Comment (bencooper222) [reply src/client/index.tsx:9]: _yike_ good catch

Comment (aadibajpai) [reply src/client/index.tsx:9]: I like the alternate syntax here bc if we reuse process.env that is sort of misleading in that process.env won't actually be available client-side

Comment (leonm1) [reply src/client/index.tsx:9]: Worth noting that TS will complain about it being undefined. You can add a globals.d.ts file with `declare var SENTRY_URL: string;` to make it happy

File: .env.template (hunk 1 above)
Comment (SamLee514) [reply .env.template:24]: yeah, rationale was that I was a dumdum and just assumed the formatting was how it was supposed to be and disregarded the README but that's fair point, will revert

File: src/client/index.tsx (hunk 2 above)
Comment (SamLee514) [reply src/client/index.tsx:9]: Oop my b tyty

File: .env.template (hunk 1 above)
Comment (bencooper222) [reply .env.template:24]: Tbh this is probably revealing of the issue of relying on .env files. If you could get #755 done, a lot of your problems would be fixed.

File: src/client/index.tsx (hunk 2 above)
Comment (SamLee514) [reply src/client/index.tsx:9]: @leonm1 regarding definePlugin, should I be expecting that the built code in dist has the actual url string in place of SENTRY_URL, or would it still show up in the code as SENTRY_URL?

Comment (SamLee514) [reply src/client/index.tsx]: jk nvm

**Comment by SamLee514** [comment]
Straight up forgot about this for a hot minute lol @leonm1 these changes look good?

**Review by leonm1 — APPROVED** [review]
LGTM

**Pull request merged**